### PR TITLE
rename lib name to hwcomposer.broxton

### PR DESCRIPTION
**Pull request by swimmingcarp**
Tracked-On: OAM-88753
Signed-off-by: yifang ma <yifangx.ma@intel.com>

### CODE REVIEW

File: tests/Android.bp [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 
 cc_test {
```
Comment (feijiang1) [inline tests/Android.bp]: why change to a totally different name?

File: Android.bp [hunk 2]
```diff
@@ -13,10 +13,10 @@
 // limitations under the License.
 
 // =====================
-// libdrmhwc_utils.a
+// libdrmhwc_utils_broxton.a
```
Comment (feijiang1) [inline Android.bp]: libdrmhwc_utils_broxton.a => libdrmhwc_utils_pri.a

**Comment by tpalli** [comment]
Is the plan to only support Broxton platform? If not, then maybe use a more generic name, typically Android HAL modules use $(TARGET_BOARD_PLATFORM) in their name.

**Comment by feijiang1** [comment]
We are planning to change hwc library name to libdrm.intel.so, due to such library can be applied to multiple platforms.

**Pull request closed without merging**